ibl_to_nwb/metadata/ecephys.yml

```diff
@@ -82,3 +82,7 @@ Ecephys:
       description: Proportion of quality metrics passed.
     - name: allen_location
       description: Brain region reference in the Allen Mouse Brain Atlas.
+    - name: uuid
+      description: Unique identifier given to each cluster after spike sorting. Enables comparison of each cluster in this .nwb file to its counterpart in ONE.
```

Comment:
```suggestion
      description: Unique identifier given to each local cluster after spike sorting. Enables comparison of each unit in this .nwb file to its counterpart cluster UUID from ONE.
```

Reply:
The emphasis in resolving nomenclature here is also important - where ONE refers to things as 'clusters', NWB refers to things as 'units' (not necessarily putative)